f5-bigip-mixin/dashboards/f5-cluster-overview.libsonnet

```diff
@@ -0,0 +1,633 @@
+    ),
+  ],
+  type: 'bargauge',
+  title: 'Top utilized virtual servers / $__interval',
+  description: 'Virtual servers with the highest traffic (inbound and outbound).',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'thresholds',
+      },
+      mappings: [],
+      min: 0,
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+        ],
+      },
+      unit: 'decbytes',
+    },
+    overrides: [],
+  },
+  options: {
+    displayMode: 'gradient',
+    minVizHeight: 10,
+    minVizWidth: 0,
+    orientation: 'horizontal',
+    reduceOptions: {
+      calcs: [
+        'lastNotNull',
+      ],
+      fields: '',
+      values: false,
+    },
+    showUnfilled: true,
+    valueMode: 'color',
+  },
+  pluginVersion: '10.2.0-60139',
+};
+
+local topLatencyVirtualServersPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'topk($k, bigip_vs_cs_mean_conn_dur{job=~"$job", instance=~"$instance"})',
+      datasource=promDatasource,
+      legendFormat='{{vs}} - {{partition}} - {{instance}}',
+      format='time_series',
+    ),
+  ],
+  type: 'bargauge',
+  title: 'Top latency virtual servers',
+  description: 'Virtual servers with the highest response times.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'thresholds',
+      },
+      mappings: [],
+      min: 0,
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+        ],
+      },
+      unit: 'ms',
+    },
+    overrides: [],
+  },
+  options: {
+    displayMode: 'gradient',
+    minVizHeight: 10,
+    minVizWidth: 0,
+    orientation: 'horizontal',
+    reduceOptions: {
+      calcs: [
+        'lastNotNull',
+      ],
+      fields: '',
+      values: false,
+    },
+    showUnfilled: true,
+    valueMode: 'color',
+  },
+  pluginVersion: '10.2.0-60139',
+};
+
+{
+  grafanaDashboards+:: {
+    'f5-cluster-overview.json':
+      dashboard.new(
+        'F5 cluster overview',
+        time_from='%s' % $._config.dashboardPeriod,
+        tags=($._config.dashboardTags),
+        timezone='%s' % $._config.dashboardTimezone,
+        refresh='%s' % $._config.dashboardRefresh,
+        description='',
+        uid=dashboardUid,
+      )
+      .addLink(grafana.link.dashboards(
+        asDropdown=false,
+        title='Other F5 dashboards',
+        includeVars=true,
+        keepTime=true,
+        tags=($._config.dashboardTags),
+      ))
+      .addTemplates(
+        [
+          template.datasource(
+            promDatasourceName,
+            'prometheus',
+            null,
+            label='Data Source',
+            refresh='load'
+          ),
+          template.new(
+            'job',
+            promDatasource,
+            'label_values(bigip_node_status_availability_state,job)',
+            label='job',
```

Comment:
don't we usually capitalize the first letter of these labels (sentence case)?

Reply:
I think I started without the linter and applied the suggestion. Guess that is why we exclude the `template-job-rule` to get it to pass.
<img width="1155" alt="Screenshot 2023-09-19 at 4 43 20 PM" src="https://github.com/grafana/jsonnet-libs/assets/13648435/d04253df-6c1b-4b23-b1f1-4f5fc0fa2ab9">

Updated